### DISCUSSION TITLE
Aside from gcc/g++, is it required to prefix 'ccache' with other toolchain commands like strip, ar, ld, objcopy, and so on?

### DISCUSSION THREAD

**Discussion by jetm** [ccache/ccache#1346] [Questions & answers] [answered]
The documentation clearly mentions the use of ccache with `gcc` and `g++`.

Does ccache should be used with the rest of the toolchain commands like `strip`, `ar`, `ld`, `objcopy` and so on? Is there any benefit?

**Comment by jrosdahl** [comment, marked as answer]
ccache currently only works for compilers. There is no benefit to try to use it for other tools.
